Formula/golang-migrate.rb

```diff
@@ -14,16 +14,14 @@ class GolangMigrate < Formula
 
   depends_on "go" => :build
 
-  def install
-    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
-    (buildpath/"src/github.com/golang-migrate/migrate").install buildpath.children
+  patch do
+    url "https://github.com/golang-migrate/migrate/commit/ea98a5bc9f772fe4747e4aac0f60ab0898ba758d.patch?full_index=1"
+    sha256 "2ee3f6233dfd05cfb500b51d4ca9172bf73705e8e25070ed3cf9cb52040377ab"
+  end
 
-    # Build and install CLI as "migrate"
-    cd "src/github.com/golang-migrate/migrate" do
-      system "make", "build-cli", "VERSION=v#{version}"
-      bin.install "cli/build/migrate.darwin-amd64" => "migrate"
-      prefix.install_metafiles
-    end
+  def install
+    system "make", "build", "VERSION=v#{version}"
```

Comment:
You'll want to use [v4.12.1](https://github.com/golang-migrate/migrate/releases/tag/v4.12.1) for the new make target

Reply:
Thanks, will update later